pom.xml

```diff
@@ -354,6 +354,7 @@
     <!-- The version greater than 1.0.0.GA is not compatible with GWT 2.8.x -->
     <!-- therefore the property is rewritten in that repository parent -->
     <version.javax.validation>2.0.1.Final</version.javax.validation>
+    <version.jakarta.validation>3.0.1</version.jakarta.validation>
```

Comment:
@rgdoliveira I would kee this aligned with the new cxf version upgrading, so 2.0.2

Reply:
+1, I updated the PR